ArduPlane/sensors.cpp

```diff
@@ -215,6 +219,10 @@ void Plane::update_sensor_status_flags(void)
         control_sensors_enabled |= MAV_SYS_STATUS_LOGGING;
     }
 
+    if (g.fs_batt_voltage != 0 || g.fs_batt_mah != 0) {
```

Comment:
Not sure, 0 was the enable value, although on reflection I think the voltage one is a float so that should change to is_zero(). I'm not sure if there is any logical reason, but I'm pretty sure you could be looking at negative voltages if you had a negative multiplier.